game_patch/object/object.cpp

```diff
@@ -187,8 +189,19 @@ CodeInjection sort_clutter_patch{
 
 FunHook<rf::VMesh*(rf::Object*, const char*, rf::VMeshType)> obj_create_mesh_hook{
     0x00489FE0,
-    [](rf::Object* objp, const char* name, rf::VMeshType type) {
-        rf::VMesh* mesh = obj_create_mesh_hook.call_target(objp, name, type);
+    [] (rf::Object* const objp, const char* name, const rf::VMeshType type) {
+        const std::input_iterator auto level = g_af_level_info_config
```

Comment:
`level`? that's a stupid name. Better `mesh_replacements` or `replacements`, because i'ts clearly not a level...